packages/data-point/lib/entity-types/entity-request/resolve.js

```diff
@@ -96,41 +96,33 @@ function inspect (acc) {
 module.exports.inspect = inspect
 
 /**
- * @param {Accumulator} acc
+ * @param {Object} options
+ * @return {Promise}
+ */
+function _requestReducer (options) {
+  return rp(options)
+}
+
+// this name will appear in the stack trace when a request fails
+Object.defineProperty(_requestReducer, 'name', {
+  value: 'request-promise#request'
+})
+
+// this function is a reducer so that we can generate
+// reducer stack traces if the request has an error
+const requestReducer = createReducer(_requestReducer)
+
+module.exports.requestReducer = requestReducer
+
+/**
+ * @param {Accumulator} accumulator
  * @param {Function} resolveReducer
  * @return {Promise<Accumulator>}
  */
-function resolveRequest (acc, resolveReducer) {
-  inspect(acc)
-  return rp(acc.options)
-    .then(result => utils.set(acc, 'value', result))
-    .catch(error => {
-      // remove auth objects from acc and error for printing to console
-      const redactedAcc = fp.set('options.auth', '[omitted]', acc)
-      const redactedError = fp.set('options.auth', '[omitted]', error)
-
-      const message = [
-        'Entity info:',
-        '\n  - Id: ',
-        _.get(redactedAcc, 'reducer.spec.id'),
-        '\n',
-        utils.inspectProperties(
-          redactedAcc,
-          ['options', 'params', 'value'],
-          '  '
-        ),
-        '\n  Request:\n',
-        utils.inspectProperties(
-          redactedError,
-          ['error', 'message', 'statusCode', 'options', 'body'],
-          '  '
-        )
-      ].join('')
-
-      // attaching to error so it can be exposed by a handler outside datapoint
-      error.message = `${error.message}\n\n${message}`
-      throw error
-    })
+function resolveRequest (accumulator, resolveReducer) {
+  inspect(accumulator)
+  const acc = utils.set(accumulator, 'value', accumulator.options)
+  return resolveReducer(acc, requestReducer)
```

Comment:
This gets rid of all the custom properties that were added to the error object here. My reasoning for this change is that I'm now adding `error#_value` and `error#_stack` properties that have a lot of the same information, and most of the other properties are just repeating data that's already available in the original error. now that we're no longer doing `console.info` with the error message (#210), it seems less important to have this code.